lib/index.js

```diff
@@ -499,6 +503,45 @@ function validateDate(date, key) {
     `"${key}" must be a UNIX timestamp or JavaScript Date.`);
 }
 
+/**
+ * Takes in an algorithm and the headers parameters from the signature.
```

Comment:
And does what? ... Perhaps something like this instead:

```suggestion
 * Ensures the given algorithm and the headers parameters from the signature
 * are compatible with one another.
```